conf/airframes/TUDELFT/tudelft_ardrone2_opticflow_stereo.xml

```diff
@@ -23,8 +23,8 @@
 
   <modules main_freq="512">
     <module name="bat_voltage_ardrone2"/>
-    <module name="cv_opticflow"/>
-    <module name="opticflow_hover"/>
+    <!--module name="cv_opticflow"/>
+    <module name="opticflow_hover"/-->
```

Comment:
this doesn't look like this has much to do with the optic flow filter of derotation.